.github/workflows/ci.yml

```diff
@@ -286,8 +286,8 @@ jobs:
       #
       - name: Run integration test with externalized ES
         run: >
-          ./mvnw -B -T 2 --no-snapshot-updates
-          -D forkCount=7
+          ./mvnw -B -T 1 --no-snapshot-updates
+          -D forkCount=1
           -D maven.javadoc.skip=true
```

Comment:
:i: I realized that we were also stressing the ES node, I saw some log saying Yellow. I just wanted to try with lower fork count if this helps to reduce the load. It did.

Instead before having test execution around 30s (and before in old setup 60s) - it was further reduced to 10 seconds ! When I run the tests with fork count 7 it needs 6-7 minutes, now it takes [~1:40](https://github.com/camunda/camunda/actions/runs/12943304681/job/36102432356?pr=27342#step:8:375). I'm a bit shocked

Reply:
OK, I wouldn't have expected that 😄

Reply:
It also probably means that it will be much "lighter" on resources when running locally. It happened to me to hit OOMs.
Great! :rocket:

Reply:
So things were slow because we were overloading ES? We discussed that before, right? :smile: